plugins/colored-man-pages/colored-man-pages.plugin.zsh

```diff
@@ -25,7 +25,7 @@ man() {
 		LESS_TERMCAP_so=$(printf "\e[1;44;33m") \
 		LESS_TERMCAP_ue=$(printf "\e[0m") \
 		LESS_TERMCAP_us=$(printf "\e[1;32m") \
-		PAGER=/usr/bin/less \
+		PAGER='/usr/bin/env less' \
```

Comment:
Thanks for spotting this! 

Let's use a builtin zsh feature though: `$commands[name]` will return the path of the command *name* if it exists (otherwise it returns blank). So here we could use that to even protect this in systems where `less` is not installed:
```zsh
PAGER="${commands[less]:-$PAGER}" \
```
This has the benefit of working also if `env` is not installed in `/usr/bin/env`.